combinedIngress/helper.py

```diff
@@ -38,12 +38,22 @@ def write_to_yaml(python_object, filepath):
         yaml.dump(python_object, file)
 
 
+def validate_dns(dns_name):
```

Comment:
we can just slugify, or like I said in the issue, use the GitLab `CI_COMMIT_REF_SLUG` which already is a slug version of the branch name.

Reply:
ahh, @pmac I'll try the slugify then.

I think the `CI_COMMIT_REF_SLUG` won't work here, since I need not just the branch that this is being run in, but all of the branches matching a naming scheme. (I'm assuming gitlab doesn't expose ref slugs for other branches in the same repo).

Reply:
That is true. Forgot you weren't just working with the pushed branch. I've used [this library for slugification](https://github.com/un33k/python-slugify) before and it's worked quite well.